packages/jsii-rosetta/lib/rosetta-translator.ts

```diff
@@ -60,6 +69,15 @@ export class RosettaTranslator {
     }
   }
 
+  public async addToCache(filename: string) {
```

Comment:
It almost feels like `addToCache` should be the way we load caches rather than `loadCache`, since it does the same thing but allows for us to add multiple tablets. Any chance you'd let me deprecate `loadCache`?

Reply:
Yep, that's fine.